Another pass at the getting started doc

liquidx opened the pull request:
Sentence casing titles.
Making the first few paragraphs less wordy.
Restructured the contents of the Javascript section.

Garbee commented on docs/getting-started.md:
```diff
@@ -1,54 +1,49 @@
 ### Creating the skeleton index.html file
 
-Now that we have a sane environment set up, let's create a simple `index.html` file, and include
+Now that we have a the environment set up, let us create a simple `index.html` file, and include
```
> an environment set up
Just one article.

>, create a simple
Drop the `let us` entirely. Docs should be written in active voice of the second person viewpoint.

liquidx replied:
Done

Garbee commented on docs/getting-started.md:
```diff
@@ -69,33 +64,37 @@ the assets needed for MDLv2. Put the following within `index.html` in the `greet
   happening on the page. As we develop our greeting app, we'll
   add in the necessary JavaScript.
-* **No automatic DOM rendering** - For all components, MDL does not render _any_ DOM elements
+* **No automatic DOM rendering** - For all components, MDLv2 does not render _any_ DOM elements
```
`MDLv2` to just `MDL`. V2 is an assignment version not the actual name of the project.

liquidx replied:
I mainly tried to make it as consistent as possible throughout the doc since it was using both MDL and MDLv2 inconsistently.

Garbee commented on docs/getting-started.md:
```diff
@@ -1,54 +1,49 @@
 The best way to learn any new technology is to get your hands dirty and build something with it, so
-that's what we'll do here! We'll be building a simple web page which lets you enter a first and/or
-last name, and then greets you using your name. You can see a finished example [here](https://plnkr.co/edit/ahd84pIgOF7OTKgavvPP?p=preview).
+that is what we will do here!  We will be building a simple greeting page which lets you enter a name and greets you as such. Here is the [finished example](https://plnkr.co/edit/ahd84pIgOF7OTKgavvPP?p=preview).
```
We > you

Second person (mentioned in another part of the review as well further down.)

liquidx replied:
Done

Garbee commented:
Overall looks good and the wording flows better in general. A few comments inline on things that shouldn't be changed or that need to be in an alternative voice.

liquidx commented:
I addressed the comments, but I think I may have made a mistake with the pull request since I'm merging in to docs/integration-with-frameworks branch instead of master that I intended to. 

I'm going to recreate this PR again. Sorry for the hassle.
